src/export/exporter.ts

```diff
@@ -3,19 +3,18 @@ import { Logger } from '@opentelemetry/api';
 import { ConsoleLogger, LogLevel } from '@opentelemetry/core';
 import { Envelope } from '../Declarations/Contracts';
 import { ConnectionStringParser } from '../utils/connectionStringParser';
-import { HttpSender } from '../platform';
+import { HttpSender, FileSystemPersist } from '../platform';
 import { DEFAULT_EXPORTER_CONFIG, AzureExporterConfig } from '../config';
-import { BaseExporter, TelemetryProcessor } from '../types';
-import { ArrayPersist } from '../platform/nodejs/persist/arrayPersist';
+import { BaseExporter, TelemetryProcessor, PersistentStorage, Sender } from '../types';
 import { isRetriable, BreezeResponse } from '../utils/breezeUtils';
 import { ENV_CONNECTION_STRING, ENV_INSTRUMENTATION_KEY } from '../Declarations/Constants';
 
 export abstract class AzureMonitorBaseExporter implements BaseExporter {
-  protected readonly _persister: ArrayPersist<Envelope[]>; // @todo: replace with FileSystemPersister
+  protected readonly _persister: PersistentStorage;
 
   protected readonly _logger: Logger;
 
-  protected readonly _sender: HttpSender;
+  protected readonly _sender: Sender;
```

Comment:
Is HttpSender still available?  rename?

Reply:
Changed this to be this interface name instead so that it is swappable to PostSender for browsers.